features/support/formatter.rb

```diff
@@ -34,7 +34,7 @@ def initialize(runtime, path_or_io, options)
 
       #
 
-      def before_features(features)
+      def before_features(_features)
```

Comment:
Since this is an implementation on a reference I vote just do `(_)` instead of the full name.  People can always look at the reference class for the implementation that they need and what arguments it accepts, they just need know we don't care about them here.